demos-europe/demosplan-core
# fix(refs DPLAN-11601): Only allow orgas with E-Mail addresses to be ivitable

### Ticket
DPLAN-11601

Orgas with no Email-address (double opt in pending) should be unselectable in the "invite orgas list".
Add "lock-checkbox-by" to the invite institution list by email, to prevent that instutions without an email adress can be invited , wich would trigger a mailsend to an undefined  address. 

To archive this the main change had to happen in demosplan-ui. See related PRs




### How to review/test


with the changes from demosplan-ui:
on the page verfahren/<procedure-id>/einstellungen/benutzer/hinzufuegen

only orgas with an email address should be selectable. (should also work with the select all toggle) 


### Linked PRs (optional)
List other PRs that are somehow connected to this and explain the connection.

- Other [PR1 #{PR-number1}](https://github.com/demos-europe/demosplan-ui/pull/878)




### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
